### PR TITLE
adding java option for gwc for geoserver-geofence

### PR DESCRIPTION
**Pull request by jeanmi151**
I had the very same issue ( https://github.com/geoserver/geofence/issues/102 ) with georchestra/geoserver/23.0.2-geofence 
Found this working solution

### CODE REVIEW

**Comment by pmauduit** [comment]
it could also relates to https://github.com/geoserver/geoserver/pull/7084

File: geoserver/webapp/src/docker/Dockerfile [hunk 1]
```diff
@@ -38,6 +38,7 @@ ENV LD_LIBRARY_PATH=/mnt/geoserver_native_libs:/opt/libjpeg-turbo/lib64
 
 CMD ["sh", "-c", "exec java -Djava.io.tmpdir=/tmp/jetty \
 -Dgeofence-ovr=file:/etc/georchestra/geoserver/geofence/geofence-datasource-ovr.properties \
+-Dgwc.context.suffix=gwc
```
Comment (pmauduit) [inline geoserver/webapp/src/docker/Dockerfile:41]: missing final '\'

Comment (jeanmi151) [reply geoserver/webapp/src/docker/Dockerfile:41]: good catch thanks

**Comment by fvanderbiest** [comment]
LGTM, please merge & backport to 23.0.x

**Comment by jeanmi151** [comment]
backported into 23.0.x too : https://github.com/georchestra/georchestra/commit/fdb4b90df4c4308eaba65cae104afb3e30ec6e34

**Pull request merged**